src/de/jost_net/JVerein/gui/view/RechnungMailView.java

```diff
@@ -84,10 +82,10 @@ public void bind() throws Exception
         DokumentationUtil.RECHNUNG, false, "question-circle.png");
```

Comment:
Das würde ich RECHNUNG_MAIL nennen und auf die alte Beschreibung verlinken.
Beim Rechnung View sollte dann RECHNUNG auf eine neue Seite in der Docu verlinkt werden.